prime-router/docs/design/proposals/0023-condition-to-code-mapping/0023-condition-to-code-mapping.md

```diff
@@ -0,0 +1,229 @@
+
+### Supplemental Condition Mapping Table
+
+The RCTC does a fairly good job of keeping up to date with LOINC and SNOMED codes and is regularly updated. It is anticipated that despite this there will be a requirement to map codes that are not present in the condition mapping table. This will be accomplished by a supplemental condition table in a similar manner to the LIVD supplemental table with the exception that the two tables will not be merged into a single table in the database. Instead the supplemental table will only be queried if there is a failure to find a match on the main table. The supplemental table will contain the same columns as the main table. If a column is not applicable it can be left blank unless it is the "Code", "Condition Name" or "Condition Code" columns.
+
+Example:
+
+| Column Name                   | Description                                   | Example                                        |
+|-------------------------------|-----------------------------------------------|------------------------------------------------|
+| Member OID                    | ValueSet Identifier                           | 2.16.840.1.114222.4.5.288                      |
+| Name                          | ValueSet Name                                 | Lab Test (PHLIP)                               |
+| Code                          | LOINC or SNOMED coded value                   | PLT1483                                        |
+| Descriptor                    | LOINC or SNOMED term descriptio               | Cabapenemase [Presence] in isolate by Carba NP |
+| Code System                   | Indicates whether code is LOINC or SNOMED     | PLT                                            |
+| Version                       | LOINC or SNOMED release version               | 2022-09-22                                     |
+| Status                        | Indicates if code is active or depracated     | Active                                         |
+| Condition Name                | Name of associated reportable condition       | Carbapenem resistant Enterobacteriaceae        |
+| Condition Code                | SONMED value associated with condition        | 712662001                                      | 
+| Condition Code System         | System used for condition code                | SNOMEDCT                                       |
+| Condition Code System Version | SNOMED version associated with condition code | 2023-03                                        |
+
+### Appending condition information to FHIR Bundle
+
+The condition information will be most useful and easy to work with if it exists in the FHIR bundle itself. Making use of the fact that FHIR allows for metadata about a resource to include tags that include a code and display value we can provide a tag to each observation resource in a FHIR bundle that describes the condition associated with that observation
```

Comment:
It is certainly an option to do it that way. The thought was that by adding the condition information onto the bundle you can then use that in other ways in the future without having to build something specifically for that use case. e.g. for reporting purposes if we create something similar to the Cov ID Result Metadata table for UP. The thinking is that the whole purpose of FHIR tags are to "identify and relate resources to process and workflow" so instead of creating something custom, if we can use existing methods why not?